### DISCUSSION TITLE
Upgrades to Big Sur

### DISCUSSION THREAD

**Discussion by EricHemingway** [kirainmoe/hasee-tongfang-macos#99] [Q&A] [answered]
Are there any special steps to installing the upgrades from Apple to Big Sur?

I don't mind trying it if no one has ever tried it yet, but if there are special steps I'd rather do it the right way :-)

**Comment by kirainmoe** [comment, marked as answer]
What means "from Apple to Big Sur" ?

If you means upgrade from macOS Catalina to macOS Big Sur, just upgrade it from System Preference Setting.

**Reply by EricHemingway** [reply]
Sorry, I mean going from Big Sur 11.4 to 11.5.1 using Apple's Built-in Software Update Utility.

**Reply by kirainmoe** [reply]
There are many people tested it. Minor version update has no problem.

**Reply by EricHemingway** [reply]
Thanks, I just wanted to make certain.    I know Catalina was like that, but was not sure about Big Sur.

Good to know, Thanks.
